### ISSUE TITLE
Metrics not being exposed even though all dependencies installed and enabled

### ISSUE THREAD

**Issue by janholbrouck** [deadtrickster/prometheus_rabbitmq_exporter#61]
I have been trying to add your exporter to my Ubuntu Xenian machine running RMQ.
I am running RabbitMQ v3.6.6 so have installed and enabled all the dependencies found here:
https://github.com/deadtrickster/prometheus_rabbitmq_exporter/releases/tag/rabbitmq-3.6.14.1 

I ran 'sudo rabbitmq-plugins list', displaying all the enabled plugins, and all the dependencies are enabled, so I would expect it to work.

![image](https://user-images.githubusercontent.com/17902310/47795981-b9775700-dd23-11e8-8b12-f29fc7eb3ca7.png)

I have not set a specific endpoint in my rabbitmq.config file, even though I have tried without result.

The problem I have is that no matter which URL I surf to, I can not seem to get the exported metrics.

The URL I use for this is UbuntuMachineIPadress:15672/api/metrics
I can perfectly access my RMQ management console with UbuntuMachineIPAdress:15672/#/ however.
I've been trying for a while to figure out why this wouldn't be working, but I can't really figure out why.

When I installed docker on my Ubuntu machine and ran the command
"docker run -p 8080:15672 deadtrickster/rabbitmq_prometheus"
I managed to get exposed metrics, but not for the local RMQ I am running on the machine.

Any help would be appreciated!

**Comment by deadtrickster** [comment]
hi, what your browser says when you open .../api/metrics? are you seeing something in the rabbitmq logs?

**Comment by janholbrouck** [comment]
Hi. thanks for the quick response. My browser says this:
"{"error":"Object Not Found","reason":"\"Not Found\"\n"}"

I have stumbled on what might be the problem though, and am trying some more things, Since I am running RabbitMQ 3.6.6 I am unsure which release I should be using. I initially used 3.6.x since I figured that referred to 3.6.6 aswel, but I am now thinking I should perhaps try one of the earlier releases, although you don't seem to have one for 3.6.6 exactly. 

My logs did not display any odd information

**Comment by janholbrouck** [comment]
So I have not yet got it to work with on of the other 2 releases (3.6.5 and 3.6.8/9), I am getting the error
{plugin_module_unloadable,"prometheus_process_collector",
                             {error,on_load_failure}}

I have tried following the steps from https://github.com/deadtrickster/prometheus_rabbitmq_exporter/issues/26
but couldn't really figure it out. I am able to remove the folder, and also able to create the folder as the rabbitmq user, so I am not sure if this is the same problem that jperville described or not.

I added a system variable in /etc/environment
RABBITMQ_PLUGINS_EXPAND_DIR="/tmp/rabbitmq-expand-plugins"

When I start the rabbitmq service this folder is also created, so all seemed okay to me.
I am however not sure if the /tmp folder has a noexec set or not, and I am unsure on how to check this.

**Comment by deadtrickster** [comment]
to check noexec run mount. also, have you tried without process collector?
________________________________

**Comment by janholbrouck** [comment]
I have tried without the process collector, then everything runs just fine but I still don't get any metrics. Browsing to /api/metrics gives me this:
{"error":"Object Not Found","reason":"\"Not Found\"\n"}

Running the "mount" command gives me this:
![image](https://user-images.githubusercontent.com/17902310/47907046-c8c4e480-de8a-11e8-8a98-24b779bc0c26.png)

I added a system variable in /etc/environment
RABBITMQ_PLUGINS_EXPAND_DIR="/tmp/rabbitmq-expand-plugins" I don't really see anything about the /tmp folder in the mount command though. then again, /tmp is a folder so can't show up there itself, can it?
